### DISCUSSION TITLE
Adding the effects of curves

### DISCUSSION THREAD

**Discussion by Billy60** [EdgeTX/edgetx#2560] [Q&A] [answered]
### Is there an existing issue for this feature request?

- [X] I have searched the existing issues

### Is your feature request related to a problem?

Could it be possible to have the possivbility to have the addition of twp curves ? That would be "diff" and 'expo" at the same time, on the Ailerons, for example.

### Describe the solution you'd like

having a second line in the Mix menus, for a 2nd curve to add up with another one: default is 'diff', could we have a line offering 'diff' added to the first one ?

**Comment by raphaelcoeffic** [comment, marked as answer]
You can compose them by using channels that are then used as the source for another. You should have a look here: https://rc-soar.com/opentx/basics/index.htm

**Reply by raphaelcoeffic** [reply]
@RC-SOAR do you think we could use the content of your wonderful web site as a base for EdgeTX documentation? (Of course with due credits)

**Reply by RC-SOAR** [reply]
Hi Raphael, thx for kind words and interest, can we continue off line? https://rc-soar.com/email.htm. or is there any way I can PM you?

**Comment by Billy60** [comment]
Thanks Raphael, this is indeed a good way. Keep on the fantastic work !

**Comment by RC-SOAR** [comment]
@Billy60 The existing features already support your example of expo + diff on the ailerons.
- set aileron expo in the aileron input (it must be done in the _input_ - not the mix - since expo must see the raw stick value [-100% to +100%]).
- set input weight to your desired rate (= aileron travel)
- set aileron mixer weights to +/-100%
- set diff in each aileron _mix_

```
INPUTS
[I]Ail: source=Aileron_stick, weight={your control rate} expo={your expo}

MIXERS
CH:RightAileron
source=[I]Ail, weight=100%, diff={your diff}

CH:LeftAileron
source=[I]Ail, weight=-100%, diff={your diff}

```

**Comment by Billy60** [comment]
Thanks Mike
Understood and will do.
Have a good day
Billy

Le sam. 15 oct. 2022, 13:22, Mike Shellim ***@***.***> a
écrit :
